dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/IdentifiableObjectUtils.java

```diff
@@ -96,7 +97,9 @@ public static String join( Collection<? extends IdentifiableObject> objects )
      */
     public static <T extends IdentifiableObject> List<String> getUids( Collection<T> objects )
     {
-        return objects != null ? objects.stream().map( o -> o.getUid() ).collect( Collectors.toList() ) : null;
+        return objects != null
```

Comment:
This gave a NPE locally, so I added a small fix. This is related to the known Hibernate issue of eventual NULL elements in the collection.